helm-charts/charts/worker-chart/templates/worker-deployment.yaml

```
@@ -0,0 +1,19 @@
+  replicas: {{ .Values.worker.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.worker.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.worker.name }}
+    spec:
+      containers:
+        - image: {{ .Values.worker.image }}
+          name: {{ .Values.worker.containerName }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)